app/enterprise/0.35-x/deployment/migrations.md

```diff
@@ -27,6 +27,17 @@ title: Migrating to 0.35
 '/portal/developers/*/password' => '/developers/*/password'
 '/portal/invite'                => '/developers/invite'
 ```
+* As a result of the switch to serverside rendering, a few portal template files need to be updated or replaced to regain full functionality:
+    1. Replace contents of partial `spec/index-vue` with:
+    https://raw.githubusercontent.com/Kong/kong-portal-templates/master/themes/default-ie11/partials/spec/index-vue.hbs
```

Comment:
do you want these to be links to the file? or are you intending for people to copy/paste this into a command?

Reply:
@Jlawlzz

Reply:
@levfishbluefish These should be links to the file